web/rainmaker/dev-packages/egov-bpa-dev/src/ui-config/screens/specs/egov-bpa/searchResource/searchResults.js

```diff
@@ -170,42 +156,16 @@ export const searchResults = {
 };
 
 const onRowClick = rowData => {
-  switch (rowData[5]) {
+  switch (rowData[3]) {
```

Comment:
Please define constants for the index with proper label and then use them. Fixes indexes access is bad

Reply:
done.